rotors_control/include/rotors_control/common.h

```diff
@@ -24,20 +24,26 @@
 #include <assert.h>
 
 #include <mav_msgs/conversions.h>
+#include <mav_msgs/default_topics.h>
 #include <nav_msgs/Odometry.h>
 
 #include "rotors_control/parameters.h"
 
 namespace rotors_control {
 
-// Default values
+// Default values, now using values from mav_comm.
```

Comment:
Can you change this comment to something, that doesn't relate to the past :)
Either remove it or add something like `the default topics (and whatever else) are taken from mav_msgs/default_topics `